paperpress.js

```diff
@@ -114,13 +115,33 @@ Paperpress.prototype._fileToItem = function (file) {
 
 	var item = {
 		title: name,
-		slug: slug
+		slug: slug,
+		content: fileContent,
+		path: null
 	}
 
-	item.path = this.pathBuilder(item, file.collectionName)
-	item.content = (fileType === '.md') ? marked.render(fileContent) : fileContent
+	if (frontMatter.test(fileContent)) {
+		var contentData = frontMatter(fileContent)
+		var attr = contentData.attributes
+
+		item = {
```

Comment:
This shouldn't be a new object, it shoudl iterate all the contentData.attributes and add the once to the original item